spanner/benchmarks_ot_test.go

```diff
@@ -0,0 +1,374 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spanner
+
+import (
+	"log"
+	"time"
+
+	"cloud.google.com/go/internal/trace"
+	metricExporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
+	traceExporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+)
+
+/*
+var muElapsedTimes sync.Mutex
+var elapsedTimes []time.Duration
```

Comment:
Is this file expected?

Reply:
Added for benchmarking on VM . Removed it now